buzzmobile/process/frame_merger/frame_merger.py

```diff
@@ -1,6 +1,49 @@
+#!/usr/bin/env python
 import cv2
 import numpy as np
 
+import datetime as dt
+import rospy
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+
+
+class Frames(object):
```

Comment:
I think WorldModel is more ambiguous than frames. The way I see it, each of the parts produce a "frame", and that gets stored in Frames, which holds all frames. Then frame_merger merges those frames. I think it's more confusing if we didn't have some form of "frames" inside the "frame_merger"